hack/release.sh

```diff
@@ -21,15 +21,15 @@ VALIDATION_TESTS="make test"
 
 source $(dirname $0)/../vendor/knative.dev/hack/release.sh
 
-PIPELINE_ARTIFACTS="pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml"
+PIPELINE_ARTIFACTS="pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml pkg/pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml"
 
 function build_release() {
   echo "🚧 🐧 Building cross platform binaries: Linux 🐧 (amd64 / arm64 / ppc64le / s390x), MacOS 🍏, and Windows 🎠"
   ETAG=${TAG} make cross-platform
 
   ARTIFACTS_TO_PUBLISH="func_darwin_amd64 func_darwin_arm64 func_linux_amd64 func_linux_arm64 func_linux_ppc64le func_linux_s390x func_windows_amd64.exe"
   ARTIFACTS_TO_PUBLISH="${ARTIFACTS_TO_PUBLISH} ${PIPELINE_ARTIFACTS}"
-  sha256sum ${ARTIFACTS_TO_PUBLISH} > checksums.txt
+  sha256sum "${ARTIFACTS_TO_PUBLISH}" > checksums.txt
```

Comment:
@lance I recall from the past iteration that quotes in this context doesn't produce the desired results. Even though shellcheck recommends the change.

Reply:
```
 @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@ Building the release @@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@ 2023-03-29 09:21:44.946754272+00:00
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
🚧 🐧 Building cross platform binaries: Linux 🐧 (amd64 / arm64 / ppc64le / s390x), MacOS 🍏, and Windows 🎠
# Removing temporary template files
go generate pkg/functions/templates_embedded.go
env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 go build -o func_darwin_arm64 -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
env CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 go build -o func_darwin_amd64 -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
env CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o func_linux_amd64 -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
env CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o func_linux_arm64 -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
env CGO_ENABLED=0 GOOS=linux GOARCH=ppc64le go build -o func_linux_ppc64le -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
env CGO_ENABLED=0 GOOS=linux GOARCH=s390x go build -o func_linux_s390x -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
env CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -o func_windows_amd64.exe -ldflags "-X main.date=20230329T092144Z -X main.vers=v20230329-6012a4af -X main.hash=6012a4af" ./cmd/func
sha256sum: 'func_darwin_amd64 func_darwin_arm64 func_linux_amd64 func_linux_arm64 func_linux_ppc64le func_linux_s390x func_windows_amd64.exe pkg/pipelines/resources/tekton/task/func-buildpacks/0.1/func-buildpacks.yaml pkg/pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml pkg/pipelines/resources/tekton/task/func-s2i/0.1/func-s2i.yaml': No such file or directory
+ EXIT_VALUE=1
+ set +o xtrace
Cleaning up after docker in docker. 
```